### PR TITLE
[IOTDB-6301] Optimize insert first

### PR DESCRIPTION
**Pull request by BUAAserein**
## Description

Add classLoader and change default timeout.

see [jira](https://issues.apache.org/jira/browse/IOTDB-6301) and [feishu doc](https://apache-iotdb.feishu.cn/docx/Oz5SdloTzotPkyxiqVWcm2c2nKg) for details.

### CODE REVIEW

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Review by HTHou — APPROVED** [review]

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java [hunk 1]
```diff
@@ -921,6 +932,27 @@ private void initSchemaEngine() {
...
+      // SourceRewriter
+      Class.forName(SourceRewriter.class.getName());
+      Class.forName(DistributionPlanContext.class.getName());
+      // LogicalPlaner
+      Class.forName(LogicalPlanVisitor.class.getName());
+      Class.forName(LogicalQueryPlan.class.getName());
+      // TsFileProcessor
+      Class.forName(TsFileProcessor.class.getName());
+    } catch (ClassNotFoundException e) {
+      logger.error("Init query engine error", e);
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java:952]: load class error？

Comment (BUAAserein) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java]: fixed

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**